ci_framework/roles/edpm_build_images/README.md

```diff
@@ -0,0 +1,35 @@
+## Role: edpm_build_images
+This role will build a edpm images, EDPM hardened uefi and ironic-python-agent images.
```

Comment:
what is the EDPM hardened uefi image?

is it a variant of ipa or a bse image it is built form?

Reply:
This is the EDPM hardened uefi image like 'overcloud-hardened-uefi-full' image that we currently used in your tripleo side it's not variant of ipa.